app/customers.py

```diff
@@ -0,0 +1,59 @@
+import datetime
+import math
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customers:
+    def __init__(self, customer: dict) -> None:
+        self.name = customer["name"]
+        self.product_cart = customer["product_cart"]
+        self.location = customer["location"]
+        self.money = customer["money"]
+        self.car = Car(customer["car"])
+        self.cost_trip = 0
+        self.cheap_shop = None
+
+    def customer_money(self) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+    def return_info(self) -> None:
+        print(f"{self.name} rides to {self.cheap_shop.name}\n")

```

Comment:
when customer rides to shop his location should be changed to shop location, same with home